ee_tests/src/support/window_manager.ts

```diff
@@ -51,6 +51,10 @@ export class WindowManager {
     }
   }
 
+  async createNewWindow() {
```

Comment:
Change this method to `openInNewWindow` so that it takes URL and opens it in a new window.